BREAKING.md

```diff
@@ -72,13 +72,17 @@ The following deprecated methods are  now removed from sequence and merge-tree.
 ### Remove ConnectionState.Connecting
 `ConnectionState.Connecting` has been removed. Migrate all usage to `ConnectionState.CatchingUp` instead.
 
+### Remove ISummaryAuthor and ISummaryCommitter
+See [#10456](https://github.com/microsoft/FluidFramework/issues/10456). `ISummaryAuthor` and`ISummaryCommitter` are deprecated and will be removed in a future release.
```

Comment:
Should this say that they are removed in this version (2.0.0), not just deprecated and will be removed in the future?

Reply:
Oh you're right. Let me make that correction in a different PR now since this one is already merged in.

Reply:
Corrected here: https://github.com/microsoft/FluidFramework/pull/10981